Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -800,27 +799,6 @@ public void Init()
 
             JsonSerializer = CreateJsonSerializer();
 
-            OnLoggerLoaded(true);
```

Comment:
Maybe ensure logging directory exists and is writable here?

Reply:
There are checks in place to see if it exists, but you will need to found out the hard way if it's writable.